dotBackendOnboarding.md

```diff
@@ -0,0 +1,227 @@
+
+### dotCMS application settings
+
+**Configure initial admin password**:
+
+* Set up the following env variables in the `~/.zshrc` file.
+	```sh
+	export DOT_INITIAL_ADMIN_PASSWORD=admin
+	```
+**Using a full starter**:
+
+* If you need to start up your instance using a **Full Starter**, just go to this file: **_core/parent/pom.xml_** , and update the `<starter.deploy.version>` property with the starter you need.
+
+	**_Example_**:
+	```xml
+	<starter.deploy.version>20231117</starter.deploy.version>
+	```
+* Alternatively, you can go directly to the following folder: **_core/dotCMS/target/starter/_** , and drop the starter’s ZIP file in there. Just make sure you rename it to **starter.zip**, as usual.
+
+* Another way is to set up the following environment variable in the `~/.zshrc` file:
+	```sh
+	DOT_STARTER_DATA_LOAD =	'https://repo.dotcms.com/artifactory/libs-release-local/com/dotcms/starter/20240213/starter-20240213.zip'
+	```
+	Finally, once the change is made (using one of the previous options), remember to **recompile** the code so that the changes will take effect, running the following command:
+	
+	| Just | Maven |
+	|--|--|
+	| `just build-no-docker`| `./mvnw clean install -DskipTests -Ddocker.skip`  |	
+
+~~**Upload a new starter**:~~
+1. Create a **_mysettings.xml_** file (can be placed in any location):
+	```xml
+	<settings>
+		<servers>
+			<server>
+				<id>dotcms-libs</id>
+				<username>{USERNAME}</username>
+				<password>{PASSWORD}</password>
+			</server>
+		</servers>
+	</settings>	
+	```
+2. Then, **run** the following command: 
+	```sh
+	./mvnw deploy:deploy-file -Durl=https://repo.dotcms.com/artifactory/libs-release-local -DrepositoryId=dotcms-libs -Dfile={PATH_OF_THE_ZIP} -DgroupId=com.dotcms -DartifactId=starter -Dversion={{empty_}timestamp} -Dpackaging=zip -s {PATH}/mysettings.xml -pl :dotcms-core
+	```
+	**_Example_**:
+	```sh
+	./mvnw  deploy:deploy-file  -Durl=[https://repo.dotcms.com/artifactory/libs-release-local](https://repo.dotcms.com/artifactory/libs-release-local)  -DrepositoryId=dotcms-libs  -Dfile=/Users/<user>/Desktop/empty_20240131.zip  -DgroupId=com.dotcms  -DartifactId=starter  -Dversion=empty_20240131  -Dpackaging=zip  -s  /Users/<user>/Documents/dotcms_master_maven/mysettings.xml  -pl  :dotcms-core
+	```
+3. Update the **_parent/pom.xml_** file with the new starter version:
+	```xml
+	<starter.deploy.version>{NEW_STARTER}</starter.deploy.version>
+	```
+
+### Local DB and ES settings
+
+**Configure a local PostgresSQL**:
+
+* Set up the following env variables in the `~/.zshrc` file.
```

Comment:
These instructions need to be cleared up.   I want to remove any need for changes to environment variables in ~/.zshrc other than the fact that this is mac only, and even then the user could be using bash.  This makes for unrepeatable builds.   These env vars I think may only be used not anyway if running Tomcat directly.    It is possible to setup maven to run against a separate external db and open search but this should not be a recommended option.   The danger is that options are set in the environment from multiple sources that conflict with the running as environment variables are global and are not contextual, e.g. running integration tests vs build etc.

Reply:
This block has been removed.